### ISSUE TITLE
SpeedDial a11y

### ISSUE THREAD

**Issue by Heydon** [iTwin/stratakit#1235]
## 1. Menu should only appear on click

State changes (between `aria-expanded` `true` and `false`) on focus. Although this is not a typical 3.2.1 On Focus violation, it is unexpected behavior. The user must give consent to change the expanded state, by click. Otherwise the state is unpredictable and confusing.
